style.css

```diff
@@ -265,89 +263,89 @@ a {
   transition: 0.4s;
 }
 
-.side_bar.collaps {
+.side_bar.collapse {
   width: 100px;
   padding: 10px;
 }
 
-.side_bar.collaps .top_bar .side_bar_btn {
+.side_bar.collapse .top_bar .side_bar_btn {
   transform: rotate(180deg);
   margin-right: 5px;
 }
 
-.side_bar.collaps .top_bar .side_bar_btn:hover {
+.side_bar.collapse .top_bar .side_bar_btn:hover {
   margin-right: 0;
 }
 
-.side_bar.collaps .top_bar span:first-child {
+.side_bar.collapse .top_bar span:first-child {
   width: 30%;
 }
 
-.side_bar.collaps .logo_group .logo_icon {
+.side_bar.collapse .logo_group .logo_icon {
   display: block;
   text-align: center;
 }
 
-.side_bar.collaps .logo_group .logo {
+.side_bar.collapse .logo_group .logo {
   display: none;
 }
 
-.side_bar.collaps .logo_group .logo-text {
+.side_bar.collapse .logo_group .logo-text {
   visibility: hidden;
 }
 
-.side_bar.collaps .menu-title {
+.side_bar.collapse .menu-title {
   color: #58595c;
   text-align: center;
 }
 
-.side_bar.collaps .menu ul li a .text {
+.side_bar.collapse .menu ul li a .text {
   position: absolute;
   left: 100%;
-  width: 120px;
+  width:max-content;
```

Comment:
This ensures the width of the element is computed from the width of the text, so it doesn't wrap onto the second line.